### PR TITLE
chore: Guardian UI README init

### PR DESCRIPTION
**Pull request by garyray-k**
README with basics for running the UI.

### CODE REVIEW

**Review by elsirion — COMMENTED** [review]
You got the braces for the markdown links the wrong way around, making link check fail.

File: guardian-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Guardian UI
+
+## Prerequisites NPM and Yarn
+
+(NPM)[https://www.npmjs.com/] - (use [nvm](https://github.com/nvm-sh/nvm))
+
+(Yarn)[https://yarnpkg.com/getting-started/installhttps://classic.yarnpkg.com/lang/en/docs/install/] - `npm install --global yarn`
```
Comment (okjodom) [inline guardian-ui/README.md:7]: we should prefer yarn installed in nix shell. See this reference https://github.com/fedimint/fedimint/blob/master/docs/gateway.md#developing-gateway-ui-aka-mintgate

Comment (garyray-k) [reply guardian-ui/README.md:7]: Should this doc be located with the other ones then?

File: guardian-ui/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Guardian UI
+
+## Prerequisites NPM and Yarn
+
+[NPM](https://www.npmjs.com/) - (use [nvm](https://github.com/nvm-sh/nvm))
+
+[Yarn](https://yarnpkg.com/getting-started/installhttps://classic.yarnpkg.com/lang/en/docs/install/) - `npm install --global yarn`
```
Comment (okjodom) [inline guardian-ui/README.md:7]: Let's prefer to use yarn installation provided by nix shell: See [this reference](https://github.com/fedimint/fedimint/blob/master/docs/gateway.md#developing-the-gateway)

File: guardian-ui/README.md (hunk 1 above)
Comment (okjodom) [reply guardian-ui/README.md]: Having both is not a problem. But the guide to enter `fedimint-ui` nix dev shell should be the same

File: guardian-ui/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (elsirion) [inline guardian-ui/README.md:7]: This link seems to be dead, but that might become irrelevant once @okjodom's concerns are addressed

Comment (okjodom) [reply guardian-ui/README.md:7]: @garyray-k let's remove this **Prerequisites** section and instead, start right away with the instructions you have listed in "Run the UI".

The goal here is to have a standard dev environment powered by nix. As such it's not a prerequisite to install npm / yarn on your dev machine. Just run the nix shell and you are good to go

File: guardian-ui/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+TODO
+
+## CI and misc.
+
+TODO
+
+## NOTE
+
+To the 🦀 devs, we're sorry for the javascript.
```
Comment (okjodom) [inline guardian-ui/README.md:33]: 😄

File: guardian-ui/README.md (hunk 3 above)
Comment (garyray-k) [reply guardian-ui/README.md]: Done.

File: guardian-ui/README.md (hunk 1 above)
Comment (garyray-k) [reply guardian-ui/README.md]: I'm going to wait to update `ui.md` once the new UI is finished. Then can update with specific instructions.

**Review by okjodom — APPROVED** [review]

**Pull request merged**